WordPress/src/main/java/org/wordpress/android/ui/activitylog/list/EventItemViewHolder.kt

```diff
@@ -36,12 +37,16 @@ class EventItemViewHolder(
         text.text = activity.description
 
         if (activity.isButtonVisible) {
-            ContextCompat.getDrawable(container.context, activity.buttonIcon.drawable)?.let { buttonIcon ->
-                val wrapDrawable = DrawableCompat.wrap(buttonIcon).mutate()
-                DrawableCompat.setTint(wrapDrawable, ContextCompat.getColor(container.context, R.color.primary_40))
-                actionButton.setImageDrawable(DrawableCompat.unwrap(wrapDrawable))
-                actionButton.visibility = View.VISIBLE
-            }
+            ContextCompat.getDrawable(container.context, activity.buttonIcon.drawable)
```

Comment:
What do you think about extracting this function so we can share it? What we want is to something like:
```
fun ImageView.setDrawableResourceWithTint(@IntRes res: Int, @ColorRes tint: Int? = null) 
```
and we'd default tint to primary color? I think this might be valuable all over the place.

Reply:
I thought I already saw an existing utility method that does this and sure thing, it was `ColorUtils.setImageResourceWithTint` :)